Readme update

dgreenhalgh opened the pull request:
Added to the README. We'll want to pull this in after we have a link to SoundStream Beta's Play Store entry.

Commit:
Added links to README

Commit:
Added pizzazz

theJenix commented on README.md:
```diff
@@ -1,9 +1,13 @@
+
+This product is currently in beta, so please keep an eye out for bugs! If you come across a scenario which you think may be a bug, please [create a new issue](https://github.com/TheLastCrusade/SoundStream/issues/new) or email us at SoundStreamHelp@gmail.com.
+
+If you have other questions, we'll be adding to our [wiki](https://github.com/TheLastCrusade/SoundStream/wiki) over time. You can suggest feature requests and see what we've got coming down the pipeline! 
```
I think we decided the wiki wouldn't be editable by people outside the core team.  How do folks suggest features at the wiki?

dgreenhalgh replied:
This is not currently the case. If we'd like to make that change, then I can tonight.

Thoughts?

theJenix replied:
I thought we decided this a few meetings ago, to avoid having to directly curate the wiki.  Suggestions would be sent to the email box, and we would add accepted ones to the wiki.  Is that not what we decided on?

dgreenhalgh replied:
Ahhh yeah. I'll make the necessary changes.

theJenix commented:
It looks like this also includes the about page changes.  Is that correct?  If so, we want to clear the issues with that first before pulling this in.

reidbaker commented:
After fixing @theJenix's comments :shipit:

reidbaker commented on README.md:
```diff
@@ -1,9 +1,13 @@
+
+If you have other questions, we'll be adding to our [wiki](https://github.com/TheLastCrusade/SoundStream/wiki) over time. You can suggest feature requests and see what we've got coming down the pipeline! 
+
+Feel free to direct any other questions to ContactLastCrusade@gmail.com!
```
Was this also going to be SoundStreamHelp@gmail.com? If not then I would say the help email is for support and the Contact email is for media request, legal, piles of money etc.

dgreenhalgh replied:
I listed both - if this doesn't read clearly, I can rephrase

reidbaker replied:
Yeah lets re phrase

dgreenhalgh commented:
The about page changes are there because I made changes on my master and then branched off of that. After we pull in the About page changes I'll fix any conflicts here.

Commit:
Readme clarifications

dgreenhalgh commented:
I added a "Contact us" section at the bottom to clear up any email address confusion

reidbaker commented:
:package: it up :shipit:

dgreenhalgh commented:
Awesome, I'll add the link to the Play Store entry and pull this in once it goes up.

Commit:
Merging in project name changes

Merge branch 'master' of https://github.com/TheLastCrusade/SoundStream into betaReadme

Commit:
Added shortened link to SoundStream Beta

Commit:
Now featuring the Hyper-Text Transfer Protocol

dgreenhalgh commented:
Added a link to the Play Store entry and pulling this in
